lib/ws-channel.js

```diff
@@ -0,0 +1,250 @@
+module.exports = WebsocketChannel;
+
+var assert = require('assert');
+var crypto = require('crypto');
+var debug = require('./debug')('ws-channel');
+var events = require('events');
+var extend = require('util')._extend;
+var util = require('util');
+var Websocket = require('ws');
+
+var REQ = 'strong-control-channel:request';
+var RSP = 'strong-control-channel:response';
+var NOT = 'strong-control-channel:notification';
+
+
+function WebsocketChannel(onRequest, onNotification) {
```

Comment:
In all my code that uses channels, I pass requests and notifications to the same callback: https://github.com/strongloop/strong-control-channel/blob/master/lib/process.js#L7-L12

Its easier to leave notifications as an API/protocol optimization on the sender's side, and ignore on the receiver's.